tchannel/tornado/util.py

```diff
@@ -90,3 +90,12 @@ def go():
     go()
 
     return all_done_future
+
+
+def num_out_pendings(conns):
```

Comment:
where does conns come from? seems like this should be a method on a connection group object